src/components/PlantGallery/PlantGallery.tsx

```diff
@@ -33,83 +15,17 @@ export default function PlantGallery({ plant }: PlantGalleryType) {
       >
         <div className="flex items-center justify-center">
           <div className="w-full max-w-xl rounded-2xl">
-            {links.length > 0 && photos[0] && (
-              <>
-                {links.map((link, index) => (
-                  <ScrollBox scrollRef={scrollRef} key={index}>
-                    <div className="h-full my-14">
-                      <ImageContainer
-                        src={link?.xl}
-                        fallback={link?.l}
-                        thumbnail={link?.xs}
-                        alt={plant.name.lat}
-                        rounded
-                      />
-                      {photos[index] && (
-                        <Credits
-                          source={link.source}
-                          owner={photos[index].owner}
-                          license={photos[index].license}
-                        />
-                      )}
-                    </div>
-                  </ScrollBox>
-                ))}
-              </>
-            )}
+            {plant.imageID.map((ID, index) => (
+              <PlantGalleryItem
+                key={index}
+                ID={ID}
+                imgAlt={plant.name.lat}
+                scrollRef={scrollRef}
+              />
+            ))}
           </div>
         </div>
       </div>
     </div>
   );
 }
-
-const ScrollBox = ({ children, scrollRef, ...rest }) => {
-  const { scrollY } = useElementScroll(scrollRef);
```

Comment:
This often throws `Error: ref provided to useScroll must be passed into a HTML element.`